geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/string/AbstractIncrByFloatIntegrationTest.java

```diff
@@ -102,50 +103,53 @@ public void testCorrectErrorIsReturned_whenKeyIsAnIncorrectType() {
 
   @Test
   public void testCorrectErrorIsReturned_whenIncrByIsInvalid() {
+    String key = "number";
     double number1 = 1.4;
-    jedis.set("number", "" + number1);
+    jedis.set(key, "" + number1);
 
-    assertThatThrownBy(() -> jedis.sendCommand(Protocol.Command.INCRBYFLOAT, "number", " a b c"))
+    assertThatThrownBy(() -> jedis.sendCommand(key, Protocol.Command.INCRBYFLOAT, key, " a b c"))
         .hasMessage("ERR value is not a valid float");
```

Comment:
There is a constant for this error message: `RedisConstants.ERROR_NOT_A_VALID_FLOAT`